### ISSUE TITLE
[Suggestion] Reduce margins in docks

### ISSUE THREAD

**Issue by chrisjlocke** [sqlitebrowser/sqlitebrowser#2299]
While looking at the dock panel for #2297, I realised the margins within the dock seem ... extra margined.  I know this has been raised before, and seems a personal preference.  However, as I've a small-ish monitor (on my laptop) its noticeable as the dock windows are quite small.

So we have this:

![image](https://user-images.githubusercontent.com/7872218/85620367-979ab880-b65b-11ea-9dea-41230d38ca1a.png)

If the top and sides were pulled in (aligned to the text and 'align' button) then it would look like this:

![image](https://user-images.githubusercontent.com/7872218/85620424-ac774c00-b65b-11ea-97df-3be658217203.png)

(sort of)

Note the remote dock spacing has been squished so the gap above the buttons is the same as the gap below them and before the grid, which is also indented a bit too much.

Unsure how this looks on other OS's, so this might just be a Windows thing, or a theme thing, etc.  Or someone else might like the gaps.  :wink:

**Comment by MKleusberg** [comment]
Yeah, Qt likes margins. I've reduced the margins from 9 pixels on each side to two pixels on each side. No margin at all looked a bit crushed on my system.

While looking at this I also noticed that there are a lot of frames. Just look at the number of vertical lines here:
![Screenshot_20200624_230706](https://user-images.githubusercontent.com/3153504/85628214-0c77ed80-b670-11ea-8428-bb6671e84c2d.png)
Maybe we should disable frames for some of these elements and reduce the margins to 0. This would make this look a lot cleaner and save even more screen space:
![Screenshot_20200624_231334](https://user-images.githubusercontent.com/3153504/85628493-85774500-b670-11ea-9edc-d648ae7494d3.png)
It does make the entire application look a bit different though. What do you think?

**Comment by justinclift** [comment]
> Maybe we should disable frames for some of these elements and reduce the margins to 0.

Well, we have the `master` branch for experimenting in and trying things out. :smile:

**Comment by chrisjlocke** [comment]
> MKleusberg added a commit that referenced this issue

Thanks for the prompt fix! I'll try this tomorrow!

**Comment by chrisjlocke** [comment]
OK, so comparing 'before' and 'after', there are still some slight adjustments.  I guess I can create a PR to tweak these?

![image](https://user-images.githubusercontent.com/7872218/85824595-19800400-b778-11ea-97e7-865209071169.png)

The 'edit database cell' grid needs moving to the left to align with the 'sql log' grid, and its width extending.  
Its top needs going up, and then its height extending, so the gap is similar to that of the SQL log grid.

Linked to this, the combobox and button on the SQL Log screen seem higher than those on the Remote tab.  
Other buttons are either 20 or 21 pixels high - these ones are 28.
